trade-finance-manager-api/src/v1/routes.js

```diff
@@ -124,4 +125,12 @@ authRouter.route('/parties/:dealId').put(validation.dealIdValidation, handleExpr
 
 authRouter.route('/bank-holidays').get(bankHolidaysController.getBankHolidays);
 
+authRouter
+  .route('/utilisation-reports/reconciliation-summary/:submissionMonth')
```

Comment:
From my previous comment on how we often split month and year, does end up looking much nicer here when it's combined, I guess we would just have query parameters and validation on them existing? At some point the submission month will mean less when it's more 'look for all open reports for today' when we allow previous report periods to be shown as well